tmk_core/common/action.c

```diff
@@ -220,6 +224,19 @@ void process_record_handler(keyrecord_t *record) {
     process_action(record, action);
 }
 
+#if defined PS2_MOUSE_ENABLE || defined POINTING_DEVICE_ENABLE
```

Comment:
Just for consistency with the rest of qmk (fully aware that these are already this exact format in other places of the file -- don't want to perpetuate it with new code):
```suggestion
#if defined(PS2_MOUSE_ENABLE) || defined(POINTING_DEVICE_ENABLE)
```